package.json

```
@@ -20,9 +20,12 @@
     "format": "pnpm run prettier:fix && yarn run lint:fix",
     "format:check": "pnpm run prettier && yarn run lint",
     "prepare": "husky && wonderland-crypto-husky-checks install",
-    "cypress:open": "cypress open",
-    "cypress:run": "cypress run",
-    "test:e2e": "pnpm cypress run",
+    "test:fork:latest": "RPC_URL=${NEXT_PUBLIC_RPC_URL:-https://ethereum-sepolia-rpc.publicnode.com/} && anvil --fork-url $RPC_URL --chain-id 11155111 --gas-limit 30000000 --gas-price 20000000000 --base-fee 1000000000 --auto-impersonate --no-storage-caching",
```

We can keep the default config unless necessary, wdyt?

```suggestion
    "test:fork:latest": "RPC_URL=${NEXT_PUBLIC_RPC_URL:-https://ethereum-sepolia-rpc.publicnode.com/} && anvil --fork-url $RPC_URL --chain-id 11155111 --no-storage-caching",
```

---

fixed [here](https://github.com/defi-wonderland/web3-nextjs-boilerplate/pull/35/commits/88fa7e7483f558711d217c1b732d7eeb80f63ade) 🙌